### PR TITLE
Make vectorize-text functions top-level

### PR DESCRIPTION
**Pull request by archmoj**
Fixes the error reported in #3379.
@plotly/plotly_js

### CODE REVIEW

**Comment by etpinard** [comment]
Thanks @archmoj !

Were you able to reproduce https://github.com/plotly/plotly.js/issues/3379 on your machine?

If not, let's wait and see if https://github.com/plotly/plotly.js/issues/3379#issuecomment-450661580 gets a reply.

**Comment by etpinard** [comment]
> Were you able to reproduce #3379 on your machine?

Ping @archmoj !

**Comment by archmoj** [comment]
No I was not able to do so. Any hint?

File: package-lock.json [hunk 1]
```diff
@@ -4716,7 +4715,7 @@
         "glslify": "^6.1.0",
         "robust-orientation": "^1.1.3",
         "split-polygon": "^1.0.0",
-        "vectorize-text": "^3.2.0"
+        "vectorize-text": "git://github.com/archmoj/vectorize-text.git#805784cb822eb29096555be6d047b45370650e9e"
```
Comment (archmoj) [inline package-lock.json]: @etpinard what do you think if I may open a PR to merge this one to Mikola's repo now?
https://github.com/mikolalysenko/vectorize-text/compare/master...archmoj:3379-vectorize-text-functions

**Comment by fuhry** [comment]
For those looking for an easy way to reproduce this, we @ZorusTech have found it reproduces under Qt Webkit including wkhtmltopdf.

**Comment by archmoj** [comment]
@etpinard OK. 
`vectorize-text` version 3.2.1 is out. Let me redo the patches using that...

**Comment by archmoj** [comment]
Closed. Now in #3474

**Pull request closed without merging**